pkg/fqdn/namemanager/cell.go

```diff
@@ -51,13 +59,29 @@ type NameManagerConfig struct {
 	StateDir string
 }
 
+type NameManagerLocalConfig struct {
+	ToFQDNsPreCache string `mapstructure:"tofqdns-pre-cache"`
+}
+
+var defaultConfig = NameManagerLocalConfig{
+	ToFQDNsPreCache: "",
+}
+
+func (def NameManagerLocalConfig) Flags(flags *pflag.FlagSet) {
+	flags.String("tofqdns-pre-cache", def.ToFQDNsPreCache, "DNS cache data at this path is preloaded on agent startup")
```

Comment:
shouldn't this be `defaultConfig.ToFQDNsPreCache`? 
def is the receiver; the mapping of flag to field happens via reflection. The value there is the default value for the flag IIRC

Reply:
I'll kill this config anyways, it's not necessary.